conda_content_trust/cli.py

```diff
@@ -352,52 +377,72 @@ def fn_addsig():
             try:
                 cct_root_signing.sign_root_metadata_dict_via_gpg(metadata, key)
             except:
-                print(F_OPTS + '\n\n--- ' + RED + 'Signing FAILED.'
-                        + F_OPTS + '  Do you have this key loaded in GPG on '
-                        'this system?')
+                print(
+                    F_OPTS
+                    + '\n\n--- '
+                    + RED
+                    + 'Signing FAILED.'
+                    + F_OPTS
+                    + '  Do you have this key loaded in GPG on '
+                    'this system?'
+                )
             else:
                 print(F_OPTS + '\n\n--- Successfully signed!  Please save.' + ENDC)
 
         else:
-            print(F_OPTS + RED + 'Unable to recognize key.  Please try again.'
-                + ENDC)
+            print(F_OPTS + RED + 'Unable to recognize key.  Please try again.' + ENDC)
         return 0
+
     def fn_remsig():
         return 0
+
     def fn_update():
         return 0
+
     def fn_adddel():
         return 0
+
     def fn_remdel():
         return 0
+
     def fn_thresh():
-        delegation = promptfor('a delegation name (one of the entries in the'
-                '\n     "delegations" dictionary in the metadata above).  '
-                'This will\n     be the delegation whose threshold number of '
-                'required keys we\n     will change.')
+        delegation = promptfor(
+            'a delegation name (one of the entries in the'
+            '\n     "delegations" dictionary in the metadata above).  '
+            'This will\n     be the delegation whose threshold number of '
+            'required keys we\n     will change.'
```

Comment:
```suggestion
            'a delegation name (one of the entries in the\n'
            '     "delegations" dictionary in the metadata above).  This will\n'
            '     be the delegation whose threshold number of required keys we\n'
            '     will change.'
```